austinbv/dino
# Adding support for HC-SR04 sensor

@rosscdh, test as follows:
1. Download this branch.
2. From the project dir, run `./bin/dino generate-sketch serial`.
3. Upload the generated dino-serial sketch to your board.
4. Customize and run the example in 'examples/hcsr04/hcsr04.rb'. It should return microsecond vlaues.

Not sure how you convert microseconds to distance for these, but that's just a matter of adding functionality to the Ruby class, so you can submit a pull against this one.
